sources/Cargo.toml

```
@@ -211,13 +211,13 @@ base64 = "0.22"
 
 [workspace.dependencies.bottlerocket-modeled-types]
 git = "https://github.com/bottlerocket-os/bottlerocket-settings-sdk"
-tag = "bottlerocket-settings-models-v0.9.0"
-version = "0.8.0"
+tag = "bottlerocket-settings-models-v0.10.0"
+version = "0.9.0"
```

I believe this should match the settings model version, and it was a miss the last time this was updated. 
```suggestion
version = "0.10.0"
```

---

@KCSesh @sky1122 it should be `0.9.0` indeed. https://github.com/bottlerocket-os/bottlerocket-settings-sdk/blob/develop/Cargo.toml#L60

---

In the current form `0.9.0` is correct, yes, but that is a bug/clerical error introduced [here](https://github.com/bottlerocket-os/bottlerocket-core-kit/commit/d17dd2620c514e04e9d8289f6639f218fdb2ec1c) when we bumped the  tag to `bottlerocket-settings-models-v0.9.0` the version bump should have happened as well. 

From what I can tell based on [Cargo](https://doc.rust-lang.org/cargo/reference/specifying-dependencies.html?highlight=tag#the-role-of-the-version-key), tag takes priority, and version is just confirming the Cargo.toml match - so I am goo to approve this.

But our next release should fix this and re-align the values.